sklearn/cluster/tests/test_hdbscan.py

```diff
@@ -531,3 +531,22 @@ def test_labelling_thresholding():
     # and the largest value is exactly MAX_LAMBDA.
     num_noise = condensed_tree["value"] < MAX_LAMBDA
     assert sum(num_noise) == sum(labels == -1)
+
+
+# TODO(1.6): Remove
+def test_hdbscan_warning_on_deprecated_algorithm_name():
```

Comment:
We can shorten `{kd, ball}tree_warning_message` variables to just `msg` since they are clear from context